app/views/group_offers/_form.html.slim

```diff
@@ -72,6 +62,31 @@
     .col-xs-12
       = f.input :schedule_details
 
+  h3= t_attr(:volunteers)
+  .row
+    .col-xs-12.col-md-4
+      = f.input :necessary_volunteers, collection: 1..10
+    - if @group_offer.volunteers.any?
+    .col-xs-12.col-md-4
+      =t_attr(:volunteers)
+      ul
+        - @group_offer.volunteers.each do |volunteer|
+          li= link_to volunteer, volunteer_path(volunteer)
+    .col-xs-12.col-md-4
+      button.btn.btn-primary(data-toggle='collapse' data-target='#volunteerBlock' type='button'  aria-expanded='false' aria-controls='volunteerBlock')
+        - if action_name ==  'new'
+          = t('.add_volunteers')
+        - else
+          = t('.edit_volunteers')
+    .col-xs-12.col-md-4
+      #volunteerBlock.collapse
```

Comment:
Not 100% sure, but I think this can be done and that would result one less nested div here.

```slim
#volunteerBlock.collapse.col-xs-12.col-md-4
```

Reply:
tried it, not really helpful, since if the chosen volunteer list is empty, it uses this row, else the row is shifted..